overlord/snapstate/backend_test.go

```diff
@@ -658,6 +658,7 @@ func (f *fakeSnappyBackend) ReadInfo(name string, si *snap.SideInfo) (*snap.Info
 		SideInfo:      *si,
 		Architectures: []string{"all"},
 		Type:          snap.TypeApp,
+		Epoch:         snap.E("1*"),
```

Comment:
should we really do this? or have it done only for a dedicated name and then have a few dedicated tests?

Reply:
I could leave it empty, which means I'd have to check for `snap.E("0")` in all the tests. I thought I might as well check for non-default while I was at it (note I left `services-snap` and `alias-snap` as the default)